test/wrapper_test.py

```diff
@@ -230,7 +230,9 @@ def testSetCredentials(self):
         self.wrapper.setCredentials('login', 'password')
         request = self._get_request(self.wrapper)
         self.assertTrue(request.has_header('Authorization'))
-        # TODO: test for header-value using some external decoder implementation
+        
+        # expected header for admin:password
```

Comment:
yupp, would be cool... just amend the commit and this PR should change on force push